scopes/envs/envs/envs.docs.md

```diff
@@ -1,33 +1,797 @@
+
+- **Less to learn**: Using a pre-configured environment extension means you don't have to get into all the details of your build tooling.
+  That's a especially important when onboarding a new developer to your team.
+
+- **Customizable and extensible**: Environments can be extended to add or override configurations.
+  Quickly add your own modifications to get an environment that best suits your needs.
+  Export your environment extension to a remote scope to have it available to all your team.
+
+- **Standardized development**: Sharing and reusing environments makes it easier to maintain consistency in development across multiple decoupled Bit workspaces.
+
+- **Easy to maintain**: Get your environment's latest updates with just a simple `bit import` command. Roll-back as easily, if needed.
+
+## CLI reference
+
+Bit environments make use of Bit's CLI to execute their different services. That means, `bit test`, for example, may execute different test runners, depending on the environment in use.
+
+#### start
+
+Runs the development serve (that includes running the Workspace UI).
+
+```shell
+// run the dev server
+$ bbit start
+```
+
+_Learn more [here](https://bit.dev/teambit/compilation/bundler)._
+
+#### build
+
+Runs the build pipeline (without tagging components with a new release version).
+
+```shell
+$ bbit build
+```
+
+_Learn more [here](https://bit.dev/teambit/pipelines/builder)._
+
+#### test
+
+Runs all tests.
+
+```shell
+$ bbit test
+```
+
+_Learn more [here](https://bit.dev/teambit/defender/tester)._
+
+#### compile
+
+Compiles all components.
+
+```shell
+$ bbit compile
+```
+
+_Learn more [here](https://bit.dev/teambit/compilation/compiler)._
+
+#### lint
+
+Get lint results for all components.
+
+```shell
+$ bbit lint
+```
+
+_Learn more [here](https://bit.dev/teambit/defender/linter)_.
+
+## Usage
+
+### Setting a default environment for the workspace
+
+Environments can only be configured using the `teambit.workspace/variants` workspace API. That means the `teambit.workspace/workspace` cannot be utilized to set an environment as the default for all components. To achieve a similar result, select all components using the `*` wildcard.
+
+For example:
+
+```json
+{
+  "teambit.workspace/variants": {
+    "*": {
+      "teambit.react/react": {}
+    }
+  }
+}
+```
+
+> A single component (with the same version) cannot use more than a single environment.
+
+### Setting multiple environments
+
+A single workspace can use different environments for different sets of components. Setting an environment on a specific group of components is done by selecting the group and applying the environment. This is done using `teambit.workspace/variants`. To learn more about using 'variant' to select components, [see here](/docs/workspace/cascading-rules)
+
+For example, to set the Node and React environments on two sets of components (selected by their directory):
```

Comment:
you should talk here about setting the `eambit.envs/envs`
consider the following ws.jsonc
```json
{
  "teambit.workspace/variants": {
    "*": {
      "teambit.react/react": {}
    },
    "components/utils": {
      "teambit.harmony/node": {}
    }
  }
}
```
With this config (which will be quite popular) every component under utils will get both node and react as aspects.
There is no guarantee in that case that the env will be used for utils is indeed node.
to solve it, you should add the `teambit.envs/envs` like this:
```json
{
  "teambit.workspace/variants": {
    "*": {
      "teambit.react/react": {}
    },
    "components/utils": {
      "teambit.harmony/node": {},
      "teambit.envs/envs": {
         "env": "teambit.harmony/node"
       }
    }
  }
}
```

Reply:
Done (see the warning in the first WS config section + the new 'troubleshooting' section)